<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>7</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;DejaVu Math TeX Gyre&quot;;</string>
    </property>
    <property name="text">
     <string>Plaque d'immatriculation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lePlaque">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;DejaVu Math TeX Gyre&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leBadge">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;DejaVu Math TeX Gyre&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leParking">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;DejaVu Math TeX Gyre&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelbadge">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>8</horstretch>
      <verstretch>8</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;DejaVu Math TeX Gyre&quot;;</string>
    </property>
    <property name="text">
     <string>Badge</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;DejaVu Math TeX Gyre&quot;;</string>
    </property>
    <property name="text">
     <string>Parking</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEffacer">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;DejaVu Math TeX Gyre&quot;;
background-color: rgb(190, 0, 0)</string>
    </property>
    <property name="text">
     <string>Effacer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEnvoyersms">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;DejaVu Math TeX Gyre&quot;;
background-color: rgb(52, 101, 164);
</string>
    </property>
    <property name="text">
     <string>Envoyer SMS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbValider">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>338</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;DejaVu Math TeX Gyre&quot;;
background-color: rgb(115, 210, 22);</string>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMessages">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;DejaVu Math TeX Gyre&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>320</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(117, 80, 123)</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>320</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(117, 80, 123)</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>320</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(117, 80, 123)</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
